io/plugins/eu.esdihumboldt.hale.io.xls/src/eu/esdihumboldt/hale/io/xls/reader/SheetSettings.java

```diff
@@ -35,6 +35,7 @@ public class SheetSettings {
 	private QName typeName;
 	private Boolean skipSheet;
 	private Integer skipLines;
+	private String dateTime;
```

Comment:
Related to the naming I think a name that at least hints at the fact that this is a format would be good.

Reply:
Done.